R/class_mrgmod.R

```diff
@@ -794,7 +794,8 @@ parin <- function(x) {
     ss_n = 500, 
     ss_fixed = FALSE, 
     interrupt = -1, 
-    etasrc = "omega"
+    etasrc = "omega", 
+    call_event = x@shlib$call_event
```

Comment:
We've saved a logical value indicating of `$EVENT` is in the model; if so, tell whether or not to call.